src/cycax/cycad/cycad_part.py

```diff
@@ -555,3 +555,27 @@ def level(
         if subtract:
             for my_side, other_side in level_tasks:
                 self.assembly.subtract(other_side, self)
+
+    def rotate(self, actions: str):
+        """
+        This can be used to rotate a part in the assembly as follows:
```

Comment:
It would be helpful to mention that there are only 90degrees counter clock wise at a time around the axis specified in the actions string. Or whatever the correct statement is.

Reply:
See PR #58